ts/components/MessageComponent.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react";
+
+import { Icon, Left, ListItem, Right, Text } from "native-base";
+import { connectStyle } from "native-base-shoutem-theme";
+import mapPropsToStyleNames from "native-base/src/utils/mapPropsToStyleNames";
+import { convertDateToWordDistance } from "../utils/convertDateToWordDistance";
+
+export type OwnProps = Readonly<{
+  serviceOrganizationName: string;
```

Comment:
Bad name, see below.

Reply:
Fixed, i have split serviceOrganizationName in : serviceOrganizationName and serviceDepartmentName.